### ISSUE TITLE
Announcement: Visual Studio 2015 Solution for Building C Interface Examples

### ISSUE THREAD

**Issue by BSalita** [WebAssembly/binaryen#500]
I've posted Visual Studio 2015 solution and project files for building the C interface examples located in WebAssembly/binaryen/test/examples. The solution builds a static library of binaryen-c.cpp for use by each example.

[Visual Studio 2015 Solution for Building C Interface Examples](https://github.com/BSalita/c-api-examples)

**Comment by kripken** [comment]
Nice!

Perhaps we should mention this in the readme?

**Comment by BSalita** [comment]
That's ok by me. However, it's a moving target as I'm busily working on additional Visual C++ features.
Yesterday the task was to build binaryen-c.cpp as a static library.
Today I was able to build binaryen-c.cpp as a DLL.
Since the DLL is now available, I was able to translate c-api-hello-world.c example into C#. Works fine. So now .Net is able to call binaryen methods. I should have c-api-kitchen-sink.c working early this week.

    On Saturday, May 14, 2016 2:51 AM, Alon Zakai <notifications@github.com> wrote:
 

 Nice!Perhaps we should mention this in the readme?—

**Comment by BSalita** [comment]
I've posted an update to the repo with source code of binaryen-c examples in C#. I see that kitchen sink blows up in the optimizer step. It does so in both original C and CS so I believe it's not my code. I'll look into it further. This work went really fast, less than half the time I estimated.
https://github.com/BSalita/c-api-examples


From README:
Visual Studio 2015 solution for building WebAssembly/binaryen/test/examples/binaryen-c files. Examples are built using test source code in bot C and C#.   
   - Builds binaryen-c.cpp into a static library.
   - Builds binaryen-c-dll.c into a dynamic library by DllExporting from the static library.
   - Builds c-api-hello-world.c (uses static library)
   - Builds c-api-kitchen-sink.c (uses static library)
   - Builds cs-api-hello-world.cs (uses dll)
   - Builds c-api-kitchen-sink.cs (uses dll)
To use, clone WebAssembly/binaryen. Be sure to edit all this repos project files (.vcxproj) to path source files at the binaryen cloned directory. 

    On Saturday, May 14, 2016 10:15 PM, Bob Salita <bsalita@yahoo.com> wrote:
 

 That's ok by me. However, it's a moving target as I'm busily working on additional Visual C++ features.
Yesterday the task was to build binaryen-c.cpp as a static library.
Today I was able to build binaryen-c.cpp as a DLL.
Since the DLL is now available, I was able to translate c-api-hello-world.c example into C#. Works fine. So now .Net is able to call binaryen methods. I should have c-api-kitchen-sink.c working early this week.

    On Saturday, May 14, 2016 2:51 AM, Alon Zakai <notifications@github.com> wrote:
 

 Nice!Perhaps we should mention this in the readme?—
